From 1b6ea684646a9a793543ca54c8d937d5fa25cc3f Mon Sep 17 00:00:00 2001
From: mattb <mattb@openmoko-trac.invalid>
Date: Thu, 11 Sep 2008 20:39:08 +0800
Subject: [PATCH] newline_after_disconnect_msg.patch

This is purely cosmetic, but annoying.

 The ar6k wifi driver does not print a newline after the informational message when disconnecting from an AP. This leads to cluttered dmesg output like the following:

 AR6000 disconnected from 00:02:cf:85:c6:08 AR6000 connected event on freq 2412 with bssid 00:02:cf:85:c6:08 listenInterval=100, beaconInterval = 100, beaconIeLen = 0 assocReqLen=27 assocRespLen =22

 What is expected, is something like:

 AR6000 disconnected from 00:02:cf:85:c6:08
 AR6000 connected event on freq 2412 with bssid 00:02:cf:85:c6:08 listenInterval=100, beaconInterval = 100, beaconIeLen = 0 assocReqLen=27 assocRespLen =22

 The upside of this is that it gives me a nice simple patch to send in as my first contribution to OpenMoko?.

 Cheers

Signed-off-by:  mattb <mattb@openmoko-trac.invalid>
---
 .../sdio/function/wlan/ar6000/ar6000/ar6000_drv.c  |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/sdio/function/wlan/ar6000/ar6000/ar6000_drv.c b/drivers/sdio/function/wlan/ar6000/ar6000/ar6000_drv.c
index 93ed978..5dd1651 100644
--- a/drivers/sdio/function/wlan/ar6000/ar6000/ar6000_drv.c
+++ b/drivers/sdio/function/wlan/ar6000/ar6000/ar6000_drv.c
@@ -2311,6 +2311,7 @@ ar6000_disconnect_event(AR_SOFTC_T *ar, A_UINT8 reason, A_UINT8 *bssid,
         A_PRINTF(" from %2.2x:%2.2x:%2.2x:%2.2x:%2.2x:%2.2x ",
                  bssid[0], bssid[1], bssid[2], bssid[3], bssid[4], bssid[5]);
     }
+    A_PRINTF("\n");
 
     AR_DEBUG_PRINTF("\nDisconnect Reason is %d", reason);
     AR_DEBUG_PRINTF("\nProtocol Reason/Status Code is %d", protocolReasonStatus);
-- 
1.5.6.5

